scripts/eurostat/health_determinants/Alcohol_Consumption/download.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This Python Script downloads the datasets in a gzip format,
+Unzips it and makes it available for further processing
+"""
+import gzip
+import os
+import urllib.request
+from input_data import input_urls
+
+
+def download_file(url) -> None:
+    """
+    Function to Download and Unzip the file provided in url
+    Args: url (str), url of the file to be downloaded as a string
+    Returns: None
+    """
```

Comment:
Please follow Google's Python Style Guide.
E.g., https://github.com/google/styleguide/blob/gh-pages/pyguide.md#383-functions-and-methods